_includes/homepage_top.html

```diff
@@ -23,7 +23,20 @@ <h1>SDSC and IceCube Center Conduct GPU Cloudburst Experiment</h1>
             </div>
           </div>
           <div class="carousel-item">
-            <img class="first-slide" src="/assets/images/des-e1521045151195.jpg" alt="Expanding universe">
+            <img class="second-slide" src="/assets/images/osg-user-school-2019.png" alt="OSG User School 2019 people">
+            <div class="container">
+              <div class="carousel-caption">
```

Comment:
The other carousel captions have class `text-left` so that the text is not center justified.   Should we be consistent?

Reply:
Two have `text-left`, six have nothing (which defaults to centered), and one has `text-right`. I think the idea was to provide flexibility in case the image makes one location best. I went with middle because most others have that and because I had no particular reason to push the text left or right.

Reply:
makes sense